action/main.ts

```diff
@@ -148,19 +146,24 @@ async function run() {
       ignoreLastBuildOnBranch: maybe(ignoreLastBuildOnBranch),
     });
 
-    const [{ url, code }] = await Promise.all([chromatic]);
+    const output = await chromatic;
```

Comment:
We don't need the `chromatic` variable here, just do `await runChromatic` directly.